app/src/main/res/layout/activity_privacy_policy.xml

```diff
@@ -16,32 +16,26 @@
   along with this program.  If not, see <https://www.gnu.org/licenses/>.
   -->
 
-<LinearLayout
+<ScrollView
```

Comment:
Please test this carefully. I've had quite some problems with ScrollViews in the past.

Reply:
The IDE tells you, that the outer `LinearLayout` is not required. If the scrolling works is not possible to test at the moment, since there are not enough items.